plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginVariantTest.kt

```diff
@@ -131,30 +128,4 @@ class SentryPluginVariantTest(
             """.trimIndent()
         )
     }
-
-    companion object {
```

Comment:
@romtsn Why did we remove that? Now we've lost the ability to run across the matrix for this test.

Reply:
it's a leftover from the past, this lives now in [here](https://github.com/getsentry/sentry-android-gradle-plugin/blob/d14f2b6f439c96bf227e73096a75b057fb365df2/plugin-build/src/test/kotlin/io/sentry/android/gradle/BaseSentryPluginTest.kt#L86-L110)

That's is still run parameterized, you can check the latest [CI run ](https://github.com/getsentry/sentry-android-gradle-plugin/actions/runs/1696777183)